digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -450,7 +450,7 @@ CommandStatus createNextCommandStatus(
                 String stoppedReason = task.getStoppedReason();
                 nextStatus.put("ecs_stopped_reason", stoppedReason);
                 throw new RuntimeException(s(
-                    "ECS Container task failed to start due to temporary AWS issues: stopCode=%s, stoppedReason=%s, containerExitCode=%d\n"
+                    "ECS Container task failed to start due to temporary AWS issues: stopCode=%s, stoppedReason=%s, containerExitCode=%s\n"
```

Comment:
I guess %d of String.format accept null value but not sure.

Reply:
at least, `%s` accepts null.